frontend/src/components/GameSelection.jsx

```diff
@@ -1,16 +1,66 @@
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import Navbar from './Navbar';
-import '../styles/GameSelection.css'
+import '../styles/GameSelection.css';
+
+import image1 from '../assets/home/image1.jpg';
+import image2 from '../assets/home/image2.jpg';
+import image3 from '../assets/home/image3.jpg';
+import image4 from '../assets/home/image4.png';
+
+import logo1 from '../assets/home/logo1.png';
+import logo2 from '../assets/home/logo2.png';
+
+const images = [image1, image2, image3, image4];
 
 const GameSelection = () => {
     const navigate = useNavigate();
+    const [currentImage, setCurrentImage] = useState(0);
+    const [searchTerm, setSearchTerm] = useState('');
+
+    useEffect(() => {
```

Comment:
does this cycle over the pictures over time?

Reply:
yes every 5 seconds